app/components/navbar.tsx

```diff
@@ -1,7 +1,8 @@
 import Link from 'next/link';
-import type { FC } from 'react';
```

Comment:
This was a mistake I think?

Reply:
Well, it seems that it is correct. We have it also for other components.
https://typescript-eslint.io/rules/consistent-type-imports/